docs/components/console/manage-clusters/cluster-capacity.md

```diff
@@ -1,70 +1,69 @@
 ---
 id: cluster-capacity
-title: Cluster capacity
-description: "Cluster capacity provides you with a high-level overview of how well a cluster is coping with and handling its current workload."
+title: Cluster load
+description: "Cluster load provides you with a high-level overview of how well a cluster is coping with and handling its current workload."
 ---
 
 import CapacityImg from './img/cluster-capacity.png';
 import CapacityPercentImg from './img/cluster-capacity-percent.png';
 
-View and manage your cluster capacity and utilization.
+Use the cluster load metric to view and manage your cluster load and utilization.
 
-## About cluster capacity
+## About cluster load
 
-Cluster capacity provides a high-level overview of how well a cluster is coping with its current workload.
+The cluster load metric provides a high-level overview of how well a cluster is coping with its current workload.
 
 - Use this information to check and monitor if a cluster is appropriately sized for its workload.
-- Cluster capacity can also be used as an indicator of cluster health. For example, a cluster running at maximum capacity can be an indicator of poor cluster responsiveness.
+- Cluster load can also be used as an indicator of cluster health. For example, a cluster running at maximum load can indicate poor cluster responsiveness.
 
-A general guideline to follow when using cluster capacity percentage as a metric is:
+A general guideline to follow when using the cluster load metric is:
 
-- **High capacity percentage**: The higher the capacity percentage value, the more likely it is that things will slow down, time out, requests will fail, and so on.
-- **Low capacity percentage**: The lower the capacity percentage value, the more you could increase the cluster workload as the cluster is probably underused.
+- **High cluster load percentage**: The higher the cluster load percentage value, the more likely it is that things will slow down, time out, requests will fail, and so on. For example, if a cluster is continually running at 95% load, this means the cluster is probably overloaded and may not be performing well.
 
-For example, if a cluster is continually running at 95%, this means it is probably overloaded and may not be performing well, but if the cluster capacity is only 5% then it can probably accept more workload and may be underused.
+- **Low cluster load percentage**: The lower the capacity percentage value, the more you could increase the cluster workload as the cluster is probably underused. For example, if a cluster load is only 5% then the cluster can probably accept more workload and may be underused.
```

Comment:
"The lower the capacity percentage value" should use load here equally to the bullet above.